ocs_ci/deployment/cnv.py

```diff
@@ -648,3 +648,135 @@ def disable_multicluster_engine(self):
             logger.error(f"Failed to disable multicluster engine\n{cmd_res.stderr}")
             return
         logger.info(cmd_res.stdout.decode("utf-8").splitlines())
+
+    def check_if_any_vm_instances(self, namespace=None):
```

Comment:
You are checking for both VM and VMI in the same function, should we update the func name accordingly?

Reply:
I have mentioned that in the doc string that we are looking for both vm and vmi